base/applications/mspaint/canvas.cpp

```diff
@@ -630,7 +630,12 @@ LRESULT CCanvasWindow::OnSetCursor(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL
     }
 
     CRect rcImage;
-    GetImageRect(rcImage);
+
+    if ((GetWindowLongW(GWL_STYLE) & WS_HSCROLL) && (GetWindowLongW(GWL_STYLE) & WS_VSCROLL))
+        GetClientRect(&rcImage);
+    else 
+        GetImageRect(rcImage);
```

Comment:
Also, since you're using && operator here, what if the window has only WS_HSCROLL or only WS_VSCROLL style?

Reply:
This issue is only visible when there are both scrollbars.